packages/entity-browser/src/components/DetailView/DetailForm.js

```diff
@@ -39,17 +39,36 @@ export const DetailForm = props => {
           fieldProps.options = {store}
         }
 
-        result.push(<div key={i} onFocus={() => props.loadRelationEntities(field.name)}>
-          <Field name={field.name} key={i} label={field.label} component={LabeledField} {...fieldProps}/></div>)
+        const isMandatoryField = fieldName => (
+          props.entityModel[fieldName]
+          && props.entityModel[fieldName].validate
+          && props.entityModel[fieldName].validate.mandatory
```

Comment:
I saw that lodash already has a function for this kind of problem.
https://lodash.com/docs/4.17.4#get
 Maybe this could be rewritten to something like this: _has(props.entityModel, fieldName + 'validate.mandatory')